### PR TITLE
Fix typos in Model Na 31-2, documentation and code comments

### PR DESCRIPTION
**Pull request by praseodym**

### CODE REVIEW

File: documentatie/softwarearchitectuur/overwegingen-talen-en-frameworks.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ Daarmee is de keuze voor Rust op basis van alle redelijke inschattingen en overw
 
 We hebben gekozen voor de volgende libraries:
 
-- De library [SQLite](https://www.sqlite.org/) die een SQL database engine implementeert. Dit is zeer populair, lichtgewicht en makkelijk in gebruik. Je hoeft bovendien niets te installeren. Het is handiger dan PostgreSQL, dat je zou moeten installeren op de computer of zou moeten meeleveren in de installer.
+- De library [SQLite](https://www.sqlite.org/) die een SQL-database engine implementeert. Dit is zeer populair, lichtgewicht en makkelijk in gebruik. Je hoeft bovendien niets te installeren. Het is handiger dan PostgreSQL, dat je zou moeten installeren op de computer of zou moeten meeleveren in de installer.
```
Comment (praseodym) [inline documentatie/softwarearchitectuur/overwegingen-talen-en-frameworks.md:149]: Don't think the added hyphen is correct here

Comment (Lionqueen94) [reply documentatie/softwarearchitectuur/overwegingen-talen-en-frameworks.md:149]: I can remove it, jetbrains checker wanted it like this, but not sure if it checked for Dutch or English in this particular instance.

Comment (Lionqueen94) [reply documentatie/softwarearchitectuur/overwegingen-talen-en-frameworks.md]: Removed in 98484fe196db12ca50abccf69ffcf7277c3f24f0

**Review by jorisleker — APPROVED** [review]

**Review by marjoleintamis — APPROVED** [review]
Nice one :)

**Pull request merged**